Patch origin: in-house
Patch status: Solaris-specific; not suitable for upstream

Allows SMF method to override Apache binary and provide additional
arguments and thus select desired MPM.

diff --git a/support/apachectl.in b/support/apachectl.in
index 3281c2e..4951991 100644
--- a/support/apachectl.in
+++ b/support/apachectl.in
@@ -42,7 +42,7 @@ ARGV="$@"
 # --------------------                              --------------------
 # 
 # the path to your httpd binary, including options if necessary
-HTTPD='@exp_sbindir@/@progname@'
+HTTPD=${HTTPD:-@exp_sbindir@/@progname@}
 #
 # pick up any necessary environment variables
 if test -f @exp_sbindir@/envvars; then
